Portfolio/index.html

```diff
@@ -1,16 +1,65 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+
+  <body style="margin: 10px; padding: 0">
+    <div top="0">
+      <a href="../index.html">Go Back</a>
+    </div>
+    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
+    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+    <h1>Pratik Shrivastava</h1>
+    <ul>
+        <li>ABOUT</li><br>
+        <li>SKILLS</li><br>
+        <li>PROJECTS</li><br>
+        <li>REVIEWS</li><br>
+    </ul>
+    <button>CONTACT ME</button>

```

Comment:
The closing <b> tag is missing the forward slash '/'. It should be written as '</b>'.